e2e/tests/components/marquee-wizard/marquee-wizard.e2e-spec.ts

```diff
@@ -58,7 +58,7 @@ describe('Marquee Wizard Tests', () => {
 
         // only the first one should actually have any content
         page.stepContents.each(async (step, idx) =>
-            expect(await step.$$('*').count()).toBe(idx === 0 ? 1 : 0));
+            expect(await step.$$('*').count()).toBe(idx === 0 ? 6 : 0));
```

Comment:
Don't really like the idea of having to update this if we happen to add another element inside the checkbox for example. Can we just assert that it's greater than zero for the first step?